content/en/real_user_monitoring/mobile_and_tv_monitoring/android/jetpack_compose_instrumentation.md

```diff
@@ -0,0 +1,226 @@
+﻿---
+title: Jetpack Compose Instrumentation
+description: Instrument Jetpack Compose manually or automatically using the Datadog Gradle Plugin.
+aliases:
+  - /real_user_monitoring/android/jetpack_compose_instrumentation/
+  - /real_user_monitoring/mobile_and_tv_monitoring/jetpack_compose_instrumentation/android
+further_reading:
+- link: https://github.com/DataDog/dd-sdk-android/tree/develop/integrations/dd-sdk-android-compose
+  tag: "Source Code"
+  text: Source code for dd-sdk-android-compose
+- link: https://github.com/DataDog/dd-sdk-android-gradle-plugin
+  tag: "Source Code"
+  text: Source code for Datadog Gradle Plugin
+- link: /real_user_monitoring
+  tag: Documentation
+  text: Explore Datadog RUM
+---
+## Overview
+If your application uses Jetpack Compose, you can instrument it manually or automatically with the Datadog Gradle Plugin. This enables Real User Monitoring (RUM) similar to what is available for Android classic Views.
+
+<div class="alert alert-info"><p>Note: The minimum supported Kotlin version is 1.9.23.</p></div>
+
+## Setup
+### Step 1 - Declare "dd-sdk-android-compose" as a dependency
+Add `dd-sdk-android-compose` dependency to each module you want to instrument. This includes the application module, any Jetpack Compose UI modules, or feature modules using Jetpack Compose.
+The minimum version of `dd-sdk-android-compose` for Jetpack Compose instrumentation is 2.21.0.
+{{< tabs >}}
+{{% tab "Groovy" %}}
+```groovy
+dependencies {
+    implementation "com.datadoghq:dd-sdk-android-compose:x.x.x"
+    //(...)
+}
+```
+{{% /tab %}}
+{{% tab "Kotlin" %}}
+```kotlin
+dependencies {
+    implementation("com.datadoghq:dd-sdk-android-compose:x.x.x")
+    //(...)
+}
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+### Step 2 - Enable actions tracking option in `RumConfiguration`
```

Comment:
I'm slightly confused as to why this is a required step.
What if customer's don't want action tracking and just want view tracking?